backend/generator.js

```diff
@@ -3,6 +3,10 @@ var exports = module.exports = {};
 var uuidV4 = require("uuid/v4");
 
 exports.executeScript = function (socket, formData) {
+  if (!isValid(formData)) {
```

Comment:
A user can always manipulate the validation on the front end,

For example, 
![image](https://user-images.githubusercontent.com/8245662/27216542-cea73ce2-5293-11e7-9e54-de0652448918.png)